lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py

```diff
@@ -178,6 +178,10 @@
         description:
             - Load balancer name.
         version_added: "2.5"
+    app_gateway:
+        description:
+            - Application gateway name.
+        version_added: "2.8"
```

Comment:
2.9?

Reply:
it will be still in 2.8, change goes into devel till feature freeze, anyway ci won't let wrong version in.